receiver/sqlserverreceiver/README.md

```diff
@@ -68,6 +73,21 @@ Example with named instance:
 
 The full list of settings exposed for this receiver are documented in [config.go](./config.go) with detailed sample configurations in [testdata/config.yaml](./testdata/config.yaml).
 
+Query sample enabled:
+```
+    receivers:
+      sqlserver:
+        collection_interval: 5s
+        username: sa
+        password: securepassword
+        server: 0.0.0.0
+        port: 1433
+        query_sample:
```

Comment:
```suggestion
        query_sample_collection:
```

Should we add a `_collection` suffix in the field name to consistent with the naming in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37958/files#diff-290e3523f19aa1515ddaa1d606e041803db5911af1ce914f56dcd3d5526c13caR90

Reply:
updated